docker-compose.yaml

```
@@ -581,11 +581,63 @@ services:
       governance-multitenant-web:
         condition: service_started
 
+  nats-1:
+    image: nats:alpine
+    ports:
+      - "8222:8222"
+      - "4222:4222"
+    command: "-c /etc/nats/nats.conf --server_name nats-1"
+    volumes:
+      - nats-data-1:/nats-data
+      - ./resources/nats-local.conf:/etc/nats/nats.conf
+    networks:
+      - nats
+    labels:
+      app: nats
+      tier: infra
+    healthcheck:
+      test: ["CMD-SHELL", 'wget -qO- localhost:8222/healthz | grep ''"status":"ok"'' || exit 1']
+      start_period: 30s
+      interval: 10s
+      timeout: 10s
+      retries: 5
+  nats-2:
+    image: nats
+    command: "-c /etc/nats/nats.conf --server_name nats-2"
+    volumes:
+      - nats-data-2:/nats-data
+      - ./resources/nats-local.conf:/etc/nats/nats.conf
+    networks:
+      - nats
+    depends_on:
+      nats-1:
+        condition: service_started
+    labels:
+      app: nats
+      tier: infra
+  nats-3:
+    image: nats
+    command: "-c /etc/nats/nats.conf --server_name nats-3"
+    volumes:
+      - nats-data-3:/nats-data
+      - ./resources/nats-local.conf:/etc/nats/nats.conf
+    networks:
+      - nats
+    depends_on:
+      nats-1:
```

I think maybe this?
```suggestion
      nats-2:
```

So `nats-2` depends on `nats-1` which depends on `nats-0`

---

They don't have to chain. The nats-1 dependency is because the first instance is whats exposed and and used from a client perspective so the dependency is just to help ensure that it startsup first. Appart from that, the dependency is not needed.